### PR TITLE
Add CD 10th birthday slice/card to the homepage

### PR DESCRIPTION
**Pull request by magdalenajadach**
Closing: https://github.com/CoderDojo/coderdojo.com/issues/91
Related to: https://github.com/RaspberryPiFoundation/digital-foundation/issues/213
Review app: https://deploy-preview-92--cdf-website.netlify.app

### CODE REVIEW

**Comment by DanielBrierton** [comment]
:heavy_check_mark: Deploy Preview for *cdf-website* ready!


:hammer: Explore the source changes: 8099f761837dd22c7cf3f56ed4898c89661423f9

:mag: Inspect the deploy log: [https://app.netlify.com/sites/cdf-website/deploys/60ec3fbc5322f80007faa6bd](https://app.netlify.com/sites/cdf-website/deploys/60ec3fbc5322f80007faa6bd?utm_source=github&utm_campaign=bot_dl)

:sunglasses: Browse the preview: [https://deploy-preview-92--cdf-website.netlify.app](https://deploy-preview-92--cdf-website.netlify.app?utm_source=github&utm_campaign=bot_dp)

File: components/index/Birthday.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      max-width: $content-width;
+      margin: 40px auto;
+      display: flex;
+    }
+
+    &__photo {
+      display: flex;
+      flex: 1;
+      margin: 20px;
+
```
Comment (glenpike) [inline components/index/Birthday.vue:35]: Image is stretched to a square in the preview.  Maybe adding `align-self: center;` on .c-birthday__photo will fix - behaves in browser and doesn't affect in responsive view.

File: assets/_variables.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@ $cd-purple-gradient: #9266a6;
 $cd-orange: #ED684A;
 $cd-orange-gradient: #EC431D;
 $cd-teal: #41BAC1;
+$foam: #D1FAFD;
```
Comment (glenpike) [inline assets/_variables.scss:10]: For consistency, this should possibly be '$cd-foam', but don't know why we prefix the colours anyway - it's a minor thing and doesn't need fixing unless you want to start doing all of the other things that need fixing too :)

**Review by glenpike — CHANGES_REQUESTED** [review]
Just needs a fix for square image.

**Review by glenpike — APPROVED** [review]
LGTM

**Pull request merged**